### ISSUE TITLE
esm-samples webpack - failed to compile

### ISSUE THREAD

**Issue by davewilton** [Esri/jsapi-resources#287]
I am reporting an issue with 

- [ ] TypeScript definitions
- [X] another resource in this repository

Download the source and unzipped (windows 10). Npm install. Npm start. ERROR in ./src/index.css
This is related to the import of the main.css. There is a css loader included so is it the fonts or images loaded by the css? there doesn't appear to be a file loader specified.

@import "~@arcgis/core/assets/esri/themes/dark/main.css";

```

> @ start C:\Temp\webpack
> webpack serve --mode development

Using @arcgis/core version: 4.18.1
i ｢wds｣: Project is running at http://localhost:3001/
i ｢wds｣: webpack output is served from 
i ｢wds｣: Content not from webpack is served from C:\Temp\webpack\dist
i ｢wdm｣: wait until bundle finished: /
× ｢wdm｣: assets by path assets/esri/ 19.6 MiB
  assets by path assets/esri/themes/ 4.52 MiB 220 assets
  assets by path assets/esri/core/ 7.5 MiB 209 assets
  assets by path assets/esri/widgets/ 4.34 MiB 87 assets
  assets by path assets/esri/images/ 413 KiB 113 assets
  assets by path assets/esri/views/ 109 KiB 9 assets
  assets by path assets/esri/symbols/patterns/*.png 3.36 KiB 8 assets    
  assets by path assets/esri/t9n/*.json 2.04 KiB 2 assets
  assets by path assets/esri/css/*.css 66 bytes 2 assets
assets by path chunks/*.js 9.51 MiB
  assets by chunk 8.6 MiB (id hint: vendors) 87 assets
  96 assets
asset index.js 4.18 MiB [emitted] (name: index)
asset ./index.html 211 bytes [emitted]
runtime modules 6.86 KiB 12 modules
orphan modules 3.46 KiB [orphan] 3 modules
modules by path ./node_modules/@arcgis/core/ 6.84 MiB 1566 modules       
modules by path ./node_modules/moment/ 672 KiB 137 modules
modules by path ./node_modules/@popperjs/core/dist/esm/ 66 KiB 51 modules
modules by path ./node_modules/webpack-dev-server/client/ 20.9 KiB 10 modules
modules by path ./node_modules/html-entities/lib/*.js 61 KiB 5 modules
modules by path ./node_modules/webpack/hot/ 1.58 KiB 3 modules
modules by path ./node_modules/url/ 37.4 KiB 3 modules
modules by path ./node_modules/querystring/*.js 4.51 KiB 3 modules
modules by path ./src/ 4.67 KiB 2 modules
modules by path ./node_modules/@esri/ 78.1 KiB
  ./node_modules/@esri/arcgis-html-sanitizer/dist/esm/index.js 64.2 KiB [built] [code generated]
  ./node_modules/@esri/calcite-colors/dist/colors.module.js 13.9 KiB [built] [code generated]
9 modules

WARNING in ./node_modules/@arcgis/core/core/workers/WorkerFallback.js 5:2264-2273
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/@arcgis/core/core/workers/workerFactory.js 5:385-420 5:2872-2873 5:2937-2938 5:2966-2967 5:4473-4474
 @ ./node_modules/@arcgis/core/core/workers/WorkerOwner.js 5:343-393 5:733-734
 @ ./node_modules/@arcgis/core/core/workers/workers.js 5:310-342 5:1436-1444
 @ ./node_modules/@arcgis/core/views/MapView.js 5:768-824 5:2027-2028
 @ ./src/index.js 6:0-49 23:17-24

WARNING in ./node_modules/@arcgis/core/core/workers/workers.js 5:863-872
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/@arcgis/core/views/MapView.js 5:768-824 5:2027-2028
 @ ./src/index.js 6:0-49 23:17-24

2 warnings have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

ERROR in ./src/index.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleParseError: Module parse failed: Unexpected character '' (1:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
    at handleParseError (C:\Temp\webpack\node_modules\webpack\lib\NormalModule.js:891:19)
    at C:\Temp\webpack\node_modules\webpack\lib\NormalModule.js:992:5
    at processResult (C:\Temp\webpack\node_modules\webpack\lib\NormalModule.js:718:11)
    at C:\Temp\webpack\node_modules\webpack\lib\NormalModule.js:778:5
    at C:\Temp\webpack\node_modules\loader-runner\lib\LoaderRunner.js:406:3
    at iterateNormalLoaders (C:\Temp\webpack\node_modules\loader-runner\lib\LoaderRunner.js:232:10)
    at Array.<anonymous> (C:\Temp\webpack\node_modules\loader-runner\lib\LoaderRunner.js:223:4)
    at runCallbacks (C:\Temp\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:27:15)
    at C:\Temp\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:200:4
    at C:\Temp\webpack\node_modules\graceful-fs\graceful-fs.js:123:16

79 ERRORS in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
webpack 5.32.0 compiled with 80 errors and 2 warnings in 12821 ms
i ｢wdm｣: Failed to compile.
```

**Comment by andygup** [comment]
@davewilton again sorry for the inconvenience, see my comments here https://github.com/Esri/jsapi-resources/pull/288#issuecomment-818788484. Yes, the loader was missing from this sample. This issue is fixed in the PR #284 along with other updates to the webpack sample.

**Comment by davewilton** [comment]
Sorry I missed that @andygup thanks for clarifying. I will close this, feel free to reject the PR. I can wait for the release as I know the fix
